api/src/main/java/org/openmrs/api/db/hibernate/HibernateConceptDAO.java

```diff
@@ -240,15 +240,15 @@ public void purgeConcept(Concept concept) throws DAOException {
 	 */
 	@Override
 	public Concept getConcept(Integer conceptId) throws DAOException {
-		return (Concept) sessionFactory.getCurrentSession().get(Concept.class, conceptId);
```

Comment:
After raising a pull request, do you take a step back and look at the changes in your pull request?

Reply:
Let me research and find out why the code automatically changes when i make a PR because i assumed that the only changes that present in the PR are the ones in my commit.